### PR TITLE
Notify users about Python packaged in F35 as RPMs

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
## Related Issues and Dependencies

Addresses: https://github.com/thoth-station/s2i-thoth/issues/231#issuecomment-1081616516
Related to: https://github.com/thoth-station/adviser/issues/1961
    - Code is based off of adviser 1961 but since our repository structure has been changed changes were needed. Additionally because the `dnf repoquery`s take a long time on a PC (2-3 hours), I will rework this into a Job which can be run on a cluster. Issue will be posted in `thoth-station/misc` for this.
Similar to: https://github.com/thoth-station/prescriptions/pull/42/files but for `f35` instead of `ubi8`

## This introduces a breaking change

- No breaking changes were introduced

## This should yield a new module release

- I am not certain but I don't think so.

## This Pull Request implements

Adding `f35` Python packages as rpms.

/cc @fridex

### CODE REVIEW

File: prescriptions/2p_/2ping/f35_rpm.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+units:
+wraps:
```
Comment (fridex) [inline prescriptions/2p_/2ping/f35_rpm.yaml:2]: Indentation looks off here. Please try running pre-commit to verify YAML files are correct.

File: prescriptions/2p_/2ping/f35_rpm.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    - name: 2pingFedora35RPMWrap
+    type: wrap
+    should_include:
+        adviser_pipeline: true
```
Comment (fridex) [inline prescriptions/2p_/2ping/f35_rpm.yaml:6]: Could you please include the operating system here? That way, we show this information only for matching environments:

```yaml
      runtime_environments:
        operating_systems:
        - name: fedora
          version: "35"
```

**Review by fridex — COMMENTED** [review]
Perfect, thanks! 💯 

Just to double-check these files pass schema validation, could you try running thoth-adviser:

```
./thoth-adviser validate-prescriptions ../prescriptions/prescriptions
```

**Comment by Gregory-Pereira** [comment]
When I ran this for the entire prescriptions directory it threw an error on file `prescriptions/_containers/quay_security.yaml`.
```
./thoth-adviser validate-prescriptions ../prescriptions/prescriptions
2022-04-01 17:05:56,693 98140 WARNING  thoth.common:344: Logging to a Sentry instance is turned off
2022-04-01 17:05:56,693 98140 INFO     thoth.common:366: Logging to rsyslog endpoint is turned off
2022-04-01 17:05:56,695 98140 INFO     thoth.adviser:248: Version: 0.47.0
2022-04-01 17:05:56,695 98140 INFO     thoth.adviser:973: Validating prescriptions in '../prescriptions/prescriptions'
2022-04-01 17:05:56,696 98140 INFO     thoth.adviser.prescription.v1.prescription:330: Using prescriptions 'thoth' release 'dev+git'
2022-04-01 17:05:56,716 98140 ERROR    thoth.adviser.prescription.v1.prescription:195: Failed to validate schema for prescription: extra keys not allowed @ data['units']['boots'][0]['metadata']
 ERROR    thoth.adviser.prescription.v1.prescription:298: Failed to load prescription from '../prescriptions/prescriptions/_containers/quay_security.yaml'
```

But when testing one of the f35_rpm.yaml files which were programatically generated it succeeds:
```
$ ./thoth-adviser validate-prescriptions ../prescriptions/prescriptions/py_/pyvo/f35_rpm.yaml
WARNING  thoth.common:344: Logging to a Sentry instance is turned off
2022-04-01 17:07:04,792 98190 INFO     thoth.common:366: Logging to rsyslog endpoint is turned off
2022-04-01 17:07:04,793 98190 INFO     thoth.adviser:248: Version: 0.47.0
2022-04-01 17:07:04,793 98190 INFO     thoth.adviser:973: Validating prescriptions in '../prescriptions/prescriptions/py_/pyvo/f35_rpm.yaml'
2022-04-01 17:07:04,794 98190 INFO     thoth.adviser:975: Prescriptions '../prescriptions/prescriptions/py_/pyvo/f35_rpm.yaml' validated successfully
count:
  boots_count: 0
  pseudonyms_count: 0
  sieves_count: 0
  steps_count: 0
  strides_count: 0
  wraps_count: 1
count_all: 1
prescriptions:
- name: UNKNOWN
  release: UNKNOWN
```

**Comment by Gregory-Pereira** [comment]
pre-commit passes locally, ready for review.

**Comment by Gregory-Pereira** [comment]
bump @fridex

**Comment by fridex** [comment]
Perfect, thanks a lot! 👍🏻 

/approve

**Pull request merged**